code/modules/power/gravitygenerator.dm

```diff
@@ -1,29 +1,24 @@
-
-//
-// Gravity Generator
-//
-
-GLOBAL_LIST_EMPTY(gravity_generators) // We will keep track of this by adding new gravity generators to the list, and keying it with the z level.
+GLOBAL_LIST_EMPTY(gravity_generators) // We will keep track of this by adding active gravity generators to the list, and keying it with the z level.
```

Comment:
```suggestion
/// Global list of all gravity generators. Keyed by the Z level
GLOBAL_LIST_EMPTY(gravity_generators) 
```

Reply:
Didn't know autodoc worked for defined stuff like that.